templates/quickstart-finance/daml/Scripts/Settlement.daml

```diff
@@ -62,44 +64,47 @@ runSettlement = do
   let
     instrumentId = Id "TOKEN"
     instrumentVersion = "0"
-    tokenInstrument = InstrumentKey
-      with
-        issuer = bank
-        depository = bank
-        id = instrumentId
-        version = instrumentVersion
+    tokenInstrument = InstrumentKey with
+      issuer = bank
+      depository = bank
+      id = instrumentId
+      version = instrumentVersion
```

Comment:
You're going against the style guide by having `with` on the same line as the type.

```suggestion
    tokenInstrument = InstrumentKey 
      with
        issuer = bank
        depository = bank
        id = instrumentId
        version = instrumentVersion
```

Reply:
I think @johan-da has been reverting to the "inline with" style in recent refactorings

Reply:
Yeah, I think we've been inconsistent with this, but not a blocker. Let's leave it as is